pkg/sqlcache/informer/listoption_indexer.go

```diff
@@ -179,6 +212,18 @@ func NewListOptionIndexer(ctx context.Context, fields [][]string, s Store, names
 		return nil, err
 	}
 
+	l.upsertEventsQuery = fmt.Sprintf(
+		`REPLACE INTO "%s_events"(rv, type, event) VALUES (?, ?, ?)`,
+		dbName,
+	)
+	l.upsertEventsStmt = l.Prepare(l.upsertEventsQuery)
+
+	l.listEventsQuery = fmt.Sprintf(listEventsAfterRVFmt, dbName)
```

Comment:
These would be more explanatory if called
`l.listEventsAfterQuery` and
`l.listEventsAfterStmt`